account-decoder/Cargo.toml

```diff
@@ -21,21 +21,27 @@ serde_derive = { workspace = true }
 serde_json = { workspace = true }
 solana-account = { workspace = true }
 solana-account-decoder-client-types = { workspace = true, features = ["zstd"] }
+solana-address-lookup-table-interface = { workspace = true, features = [
+    "bincode",
+    "bytemuck",
+] }
```

Comment:
how are you checking which features are needed for a specific crate without this being clouded by workspace feature resolution?

Reply:
I just run `cargo check` inside the account-decoder dir so I don't get workspace features

Reply:
I'm not able to reproduce that behavior. `cargo check` inside `account-decoder` doesn't fail for me even after I remove these necessary features from the dependency. I haven't been able to find a good way to check features crate by crate so I think we can just continue doing a best effort.